src/services/rpc-service.ts

```diff
@@ -46,41 +40,29 @@ export class RPCService {
     const fastest = await Promise.race(successfulPromises);
 
     if (fastest.success) {
-      latencies[`${fastest.rpcUrl}_${networkId}`] = fastest.duration;
+      latencies[`${fastest.rpcUrl}_${networkId}_`] = fastest.duration;
     }
 
-    try {
-      const allResults = await Promise.allSettled(successfulPromises);
+    const allResults = await Promise.allSettled(successfulPromises);
 
-      allResults.forEach((result) => {
-        if (result.status === "fulfilled" && (result.value as PromiseResult).success) {
-          latencies[`${(result.value as PromiseResult).rpcUrl}_${networkId}`] = (result.value as PromiseResult).duration;
-        } else if (result.status === "fulfilled") {
-          const fulfilledResult = result.value as PromiseResult;
-          const index = runtimeRpcs.indexOf(fulfilledResult.rpcUrl);
-          if (index > -1) {
-            runtimeRpcs.splice(index, 1);
-          }
+    allResults.forEach((result) => {
+      if (result.status === "fulfilled" && (result.value as PromiseResult).success) {
+        latencies[`${(result.value as PromiseResult).rpcUrl}_${networkId}_`] = (result.value as PromiseResult).duration;
+      } else if (result.status === "fulfilled") {
+        const fulfilledResult = result.value as PromiseResult;
+        const index = runtimeRpcs.indexOf(fulfilledResult.rpcUrl);
+        if (index > -1) {
+          runtimeRpcs.splice(index, 1);
         }
-      });
-
-      StorageService.setLatencies(env, latencies);
+      }
+    });
 
-      return { latencies, runtimeRpcs };
-    } catch (err) {
-      console.error("[RPCService] Failed to test RPC performance");
-    }
     return { latencies, runtimeRpcs };
   }
-
-  static async findFastestRpc(latencies: Record<string, number>, networkId: number): Promise<string> {
-    if (Object.keys(latencies).length === 0) {
-      console.error("[RPCService] Latencies object is empty");
-    }
-
+  static async findFastestRpc(latencies: Record<string, number>, networkId: number): Promise<string | null> {
     try {
       const validLatencies: Record<string, number> = Object.entries(latencies)
-        .filter(([key]) => key.endsWith(`_${networkId}`))
+        .filter(([key]) => key.endsWith(`_${networkId}_`))
```

Comment:
It makes more sense to put the network ID in the beginning for alphabetical sorting